lib/process/model/process/process_type.ex

```diff
@@ -7,7 +7,7 @@ defprotocol Helix.Process.Model.Process.ProcessType do
   @spec dynamic_resources(t) :: [resource]
   def dynamic_resources(data)
 
-  @spec event(t, Process.t, :created | :completed) :: [Helix.Event.t]
+  @spec event(t, Process.t, :created | :completed) :: [struct]
```

Comment:
Also, naturally, this problem won't happen as soon as we add that type on HELF as helf won't be recompiled on every event handler added :stuck_out_tongue: